src/backend/src/prisma/schema.prisma

```diff
@@ -122,14 +124,30 @@ model Change_Request {
 }
 
 model Scope_CR {
-  scopeCrId       Int            @id @default(autoincrement())
-  changeRequestId Int            @unique
-  changeRequest   Change_Request @relation(fields: [changeRequestId], references: [crId])
-  what            String
-  why             Scope_CR_Why[]
-  scopeImpact     String
-  timelineImpact  Int
-  budgetImpact    Int
+  scopeCrId        Int                @id @default(autoincrement())
+  changeRequestId  Int                @unique
+  changeRequest    Change_Request     @relation(fields: [changeRequestId], references: [crId])
+  what             String
+  why              Scope_CR_Why[]
+  scopeImpact      String
+  timelineImpact   Int
+  budgetImpact     Int
+  ProposedSolution ProposedSolution[]
+}
+
+model ProposedSolution {
+  proposedSolutionId String   @id @default(uuid())
+  description        String
+  timelineImpact     Int
+  budgetImpact       Int
+  scope              String
+  changeRequest      Scope_CR @relation(fields: [changeRequestId], references: [scopeCrId])
+  changeRequestId    Int
+  createdByUserId    Int
+  createdBy          User     @relation(name: "createdBy", fields: [createdByUserId], references: [userId])
+  dateCreated        Date     @default(now())
+  approvedBy         User?    @relation(name: "approvedBy", fields: [approvedByUserId], references: [userId])
+  approvedByUserId   Int?
```

Comment:
I was thinking of having an optional field for "approvedSolution" on the CR that is the ID of the solution which is approved. What do you think? Otherwise I think maybe a Boolean on the proposed solutions for approved since person approving should already be recorded by who reviewed the cr.

Reply:
is there ever a world where we have multiple proposed solutions? because doing it like this lets that world happen which is the only reason i think this could be better. If we are 100% against that forever then i agree it should be in the change request

Reply:
after discussion, i still prefer the boolean on the proposed solutions. i just think it's nicer

Reply:
no, I don't think that would make sense. see slack thread: https://nu-electric-racing.slack.com/archives/C02U5TKHLER/p1661812958854149